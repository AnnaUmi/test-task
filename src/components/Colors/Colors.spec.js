import React from 'react';
import 'jest-styled-components';
import { render } from 'react-testing-library';

import { Colors } from '.';

describe('Colors', () => {
  it('should render colors', () => {
    const { container } = render(<Colors />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
