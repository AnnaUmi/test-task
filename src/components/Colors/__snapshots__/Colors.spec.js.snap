// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors should render colors 1`] = `
<div>
  <h3>
    Brand Colors
  </h3>
  <div
    class="sc-epnACN jrRFwv"
  >
    <div>
      <span>
        Azure
      </span>
      <div
        class="sc-hXRMBi kKPvGg"
        color="rgba(2, 108, 223, 1)"
      />
    </div>
    <div>
      <span>
        Pearl Black
      </span>
      <div
        class="sc-hXRMBi hpgHNn"
        color="#1F262D"
      />
    </div>
  </div>
</div>
`;
